proto/utxorpc/v1alpha/cardano/cardano.proto

```diff
@@ -16,7 +16,9 @@ enum RedeemerPurpose {
 // Redeemer information for a Plutus script.
 message Redeemer {
   RedeemerPurpose purpose = 1; // Purpose of the redeemer.
-  PlutusData payload = 2; // Plutus data associated with the redeemer.
+  uint32 index = 2; // Index of the redeemer.
+  PlutusData payload = 3; // Plutus data associated with the redeemer.
```

Comment:
@Mercurial can we keep the `payload` as tag `2`?
In this way we don't break compatibility.